base_layer/core/src/chain_storage/blockchain_database.rs

```diff
@@ -52,7 +53,7 @@ use tari_mmr::{Hash, MerkleCheckPoint, MerkleProof, MutableMmrLeafNodes};
 
 const LOG_TARGET: &str = "c::cs::database";
 
-#[derive(Clone, Debug, PartialEq)]
+#[derive(Clone, Debug, PartialEq, Display)]
```

Comment:
If all the blocks are displayed in full in case of ChainReorg, this is not a great approach unf.